optimum/exporters/openvino/__main__.py

```diff
@@ -200,6 +201,9 @@ def main_export(
     """
     from optimum.exporters.openvino.convert import export_from_model
 
+    from ...intel.openvino.configuration import _GPTOSSQuantizationConfig
+    from ...intel.openvino.utils import TemporaryDirectory
```

Comment:
if possible would replace with absolute imports  
```suggestion
    from optimum.intel.openvino.configuration import _GPTOSSQuantizationConfig
    from optimum.intel.openvino.utils import TemporaryDirectory
```